<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptImportExport</class>
 <widget class="QWidget" name="ScriptImportExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>750</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>相机→电脑</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>280</width>
       <height>150</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>30</y>
        <width>51</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>电脑IP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ExportIP">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.0.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExport">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listScriptFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>400</width>
       <height>621</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShell">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Shell</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPython2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Python2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLua">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Lua</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPython3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Python3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>20</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>导出脚本</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>电脑→相机</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>400</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="lineEdit_ImportIP">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>280</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.0.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>电脑IP:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnImport">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>导入</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ImportName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>280</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>脚本名称:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonShell">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Shell</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonLua">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Lua</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonPython3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Python3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonPython2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Python2</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>导入脚本</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>相机→U盘</string>
    </attribute>
    <widget class="QPushButton" name="btnPython2_U">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Python2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPython3_U">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Python3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLua_U">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Lua</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShell_U">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Shell</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listScriptFiles_U">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>401</width>
       <height>651</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExport_U">
     <property name="geometry">
      <rect>
       <x>432</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>U盘→相机</string>
    </attribute>
    <widget class="QPushButton" name="btnImport_u_import">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPython2_u_import">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Python2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPython3_u_import">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Python3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLua_u_import">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Lua</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShell_u_import">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Shell</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listScriptFiles_u_import">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>400</width>
       <height>621</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
